submissions/cuteman18/js-dom/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS practice</title>
+    <link rel="stylesheet" type="text/css" href="app.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <nav>
+      <ul class="navList">
+        <li class="inscription"><a href="#" class="ref">About</a></li>
+        <li class="inscriptionDeviate"><a href="https://youtu.be/xdYYN-4ttDg" class="ref" target="_blank">Hobby</a></li>
+        <li class="inscription"><a href="#" class="ref">Mail</a></li>
+        <li class="inscription"><a href="#" class="ref">Images</a></li>
+        <li class="inscription">
+          <input type="checkbox" id="drop" class="dropDown"/><label for="drop" class="holder"></label>
+          <div class="menu">
+            <ul>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/calendar.png" alt="calendar"/></span><span class="iconText">calendar</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/mail.png" alt="mail"/></span><span class="iconText">mail</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/news.png" alt="news"/></span><span class="iconText">news</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/maps.png" alt="maps"/></span><span class="iconText">maps</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/files.png" alt="files"/></span><span class="iconText">files</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/contacts.png" alt="contacts"/></span><span class="iconText">contacts</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/photos.png" alt="photos"/></span><span class="iconText">photo</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/videos.png" alt="videos"/></span><span class="iconText">video</span></a></li>
+              <li class="icons"><a href="#" class="ref"><span class="icon"><img class="imgStyle" src="images/search.png" alt="search"/></span><span class="iconText">search</span></a></li>
+              <hr>
+              <input type="checkbox" class="button" id="extra"/><label for="extra" class="more">m o r e</label>
```

Comment:
Avoid adding visual styling in HTML.
HTML - for markup.
CSS - for visual representing.
I mean when another person change the text in this label spaces between letter will be saved?)

Reply:
Add styles on `:focus` state.

Reply:
> Avoid adding visual styling in HTML.
> HTML - for markup.
> CSS - for visual representing.

@A-Ostrovnyy. Do you mean moving image source to html?  I think it should stay inside html file -  for example here
(https://smallbusiness.chron.com/add-image-css-file-51301.html)  we can see that  :

"Web pages normally add images using the inline HTML "img" tag. CSS coding usually doesn't set an image's source because CSS controls design rather than content. "

Reply:
I mean text in `label`. Avoid formatting text in HTML using spaces or uppercase letters etc.